operators/assisted-service-operator/assisted-service.package.yaml

```diff
@@ -3,7 +3,7 @@ channels:
     name: alpha
   - currentCSV: assisted-service-operator.v0.0.8
     name: ocm-2.3
-  - currentCSV: assisted-service-operator.v0.1.0
+  - currentCSV: assisted-service-operator.v0.1.1
     name: ocm-2.4
```

Comment:
This is using stuff from ocpmetal/ so this is off

Reply:
good catch!